content/en/docs/concepts/configuration/secret.md

```diff
@@ -911,20 +926,14 @@ data:
   extra: YmFyCg==
 ```
 
-When creating a `Pod`, Kubernetes automatically finds or creates a service account
```

Comment:
It doesn’t automatically create a ServiceAccount any more?

Reply:
The Pod bits just aren't relevant to the docs of this secret type any more

Reply:
Fair enough.